### PR TITLE
second iteration of fixes, making it a bit cleaner

### PR DESCRIPTION
**Pull request by venam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,36 +2,37 @@
...
-your trivial computer system into a powerful, useful and beautiful
+your trivial computer system into a powerful, useful, and beautiful
 place.
 
-Some things to consider, however:
+Some things to take in consideration before opening this book:
 
-* This is a book on how to understand the parts of your system,
-and use them to customize it as you please.
+* This book is on how to understand the parts of your system,
```
Comment (feroldi) [inline README.md]: isn't this strange to read?

File: README.md [hunk 2]
```diff
@@ -2,36 +2,37 @@
...
-is wasteful. Although GUIs are important to a lot of
-people, using the terminal is a good experience for those who opt
-out of using it. This leads you to understand why we use it so much.
-One reason was already mentioned, let's rephrase for
-clarity: it's a standardized place
-on all systems; if it is POSIX compliant, then I can use my CLI knowledge
+No knowledge is wasteful, this book will teach you the wisdom of unix systems.
+Although GUIs may seem important to a wide array of individuals,
+using the terminal is even more of a valuable experience for those who opt
+out of using it. Learning to serpentine your way through them will reveal
```
Comment (feroldi) [inline README.md]: I don't agree on the use of neat words here. The more this book uses simpler words, the more understandable it becomes.

**Review by feroldi — CHANGES_REQUESTED** [review]
The fixes are okay, but consider using simpler words and simpler sentences.

File: README.md (hunk 1 above)
Comment (venam) [reply README.md]: You're right on that one.

File: README.md (hunk 2 above)
Comment (venam) [reply README.md]: Swap "serpentine" by "find" and it's good to go.

**Comment by venam** [comment]
This should do it now.

**Comment by feroldi** [comment]
okay, i'll be pulling by the time i get to my machine, thanks.

**Pull request merged**